packages/atomic/src/components/atomic-result-template/atomic-result-template.tsx

```diff
@@ -118,37 +136,25 @@ export class AtomicResultTemplate {
   }
 
   private getContent() {
-    // TODO: reorganise
-    const childTemplatesBySection: HTMLAtomicResultChildrenTemplateElement[][] =
-      [];
-    const content = this.getTemplateElement().content;
-    const childrenTemplates = content.querySelectorAll(
-      'atomic-result-children-template'
+    return this.getTemplateElement().content;
+  }
+
+  public async initialize() {
```

Comment:
in `initialization-utils.ts`, you will need to edit the function that calls `initialize` to await it.